spec/fabricators/category_fabricator.rb

```diff
@@ -1,3 +1,3 @@
 Fabricator(:category) do
-  name { Faker::Lorem.words(1) }
```

Comment:
Btw the other way how to do this except what we mentioned in chatroom is using a `#join` on Array. Check ruby docs for the use.